src/systemd/sd-json.h

```diff
@@ -341,6 +342,10 @@ int sd_json_variant_unhex(sd_json_variant *v, void **ret, size_t *ret_size);
 const char* sd_json_variant_type_to_string(sd_json_variant_type_t t);
 sd_json_variant_type_t sd_json_variant_type_from_string(const char *s);
 
+static inline bool sd_json_enabled(sd_json_format_flags_t flags) {
```

Comment:
`sd_json_enabled()` -> `sd_json_format_enabled()` is better?

Also, please use `__extension__` attribute, or use `int` rather than `bool`.

Reply:
I changed it like this. I also added `const` annotation here, and changed some other places to use `const` instead of `pure`. PTAL.